src/main/java/com/googlesps/feedon/data/Donation.java

```diff
@@ -0,0 +1,37 @@
+// Class for donation offer made by restaurant
```

Comment:
This comment should be before class declaration and in the form of java doc

Reply:
Okay sure, I'll make that change.